lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -223,9 +223,10 @@ def store(payment, options = {})
 
             post[:default_card] = r.params['id'] if options[:set_default] && r.success? && !r.params['id'].blank?
 
-            r.process { update_customer(options[:customer], post) } if post.count > 0
+            r.process { update_customer(options[:customer], post.merge(expand: [:sources])) } if post.count > 0
```

Comment:
@jherreraa Nice, this fixes the errors. Though, is there a reason `expand: [:sources]` is being set within the `process` call here, instead of being added on its own line, similar to below? If not, I'd suggest doing the same here for consistency and readability.

Reply:
@curiousepic, We tried the solution the way you mentioned, nevertheless that solution causes new failures (multiple invokes for methods not allowed in unit tests), so that is the reason to solve the issue in the way you see now.

Reply:
Ah, of course! Sorry for running in circles. I don't really see a different solution, so I'm ok with this.